lib/util/npm-util.js

```diff
@@ -34,6 +36,8 @@ function findPackageJson(startDir) {
         if (!fs.existsSync(pkgFile) || !fs.statSync(pkgFile).isFile()) {
             dir = path.join(dir, "..");
             continue;
+        } else if (fs.existsSync(path.join(dir, "yarn.lock"))) {
```

Comment:
I'm 👍 to @not-an-aardvark , Addtional, for a little users, when npm not found and yarn installed, using yarn also make sense.